### DISCUSSION TITLE
sort legend based on influxdb tag

### DISCUSSION THREAD

**Discussion by nickper** [grafana/grafana#48109] [Dashboards, Panels & Visualizations]
Hi,

I have a case where I want to sort the legend with ip addresses based on another tag from the query. 
I can think of 2 options.
- Add the option to add custom legend values from which can be sorted.
- Or create a new option for a standard filter option via a custom tag (which is not added to legend values)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
